remote-workstations/new-agent-vm/rhel-standard-agent.json

```diff
@@ -149,11 +155,8 @@
         }
     },
     "variables": {
-
         "apiVersion": "2015-06-15",
         "nicName": "[concat(parameters('dnsLabelPrefix'),'Nic')]",
-        "enableAutoShutdown" : false,
```

Comment:
yeah this is always enabled on Linux and isn't controlled from here?

Reply:
this didn't get used anywhere